<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChainMenu</class>
 <widget class="QWidget" name="ChainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>哈希表的查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>75</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>270</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>哈希表的插入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BuildButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>130</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>哈希表的建立</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>-1</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>链地址法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>340</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>哈希表的删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
